docs/UserGuide.md

```diff
@@ -305,7 +305,11 @@ Format: `list -c`
 ### Filtering client profiles: `filter -c`
 
 Returns a filtered list of clients according to specified tags.
+
+TAG have to be fully specified (i.e. no partial tags like 'high' instead of 'high-end')
```

Comment:
Good spot here by specifying this in the UG. Being able to search using partial tags is a good-to-have feature, but its too late to implement that.

Perhaps an improvement for this PR is to make the title a bit clearer (or do this in a separate PR), since this part has nothing to do with negative inputs for range command